Marlin/src/inc/Conditionals_LCD.h

```diff
@@ -213,7 +213,7 @@
     #define LCD_PROGRESS_BAR
   #endif
   #if ENABLED(TFTGLCD_PANEL_I2C)
-    #define LCD_I2C_ADDRESS           0x27  // Must be equal to panel's I2C slave addres
+    #define LCD_I2C_ADDRESS           0x33  // Must be 0x33 for STM32 main boards and equal to panel's I2C slave addres
```

Comment:
When `TFTGLCD_PANEL_I2C` is used on other platforms, what should its i2c address be?